go/tricorder/api.go

```diff
@@ -25,11 +26,30 @@ var (
 // read the same data from that region.
 type Region region
 
-// NewRegion creates a new region with a particular update function
+var (
+	// The default region. Its update function does nothing and returns
+	// the current system time.
+	DefaultRegion *Region
```

Comment:
Is this initialised somewhere before the application can use it? If not, what prevents a nil-pointer dereference panic?